File: UnionPressOnSharp/UnionPressOnSharp/Forms/Models/PeriodModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace UnionPressOnSharp.Forms.Models
{
    public class PeriodModel
    {
        private string srokSubscribe;
        private string sell;
        private string transferChoice;
        private string popularity;


        [DisplayName("Срок подписки")]
        [Required(ErrorMessage = "поле Срок подписки обязательное")]
        [StringLength(40, MinimumLength = 2, ErrorMessage = "длина от 2 до 40 симовлов")]
        public string SrokSubscribe
        {
            get { return srokSubscribe; }
            set { srokSubscribe = value; }
        }
        [DisplayName("Стоимость")]
        [Required(ErrorMessage = "поле Стоимость обязательное")]
        [StringLength(40, MinimumLength = 2, ErrorMessage = "длина от 2 до 40 симовлов")]
        public string Sell
        {
            get { return sell; }
            set { sell = value; }
        }
        [DisplayName("Способ доставки")]
        [Required(ErrorMessage = "поле Способ доставки обязательное")]
        [StringLength(40, MinimumLength = 2, ErrorMessage = "длина от 2 до 40 симовлов")]
        public string TransferChoice
        {
            get { return transferChoice; }
            set { transferChoice = value; }
        }
        [DisplayName("Популярность %")]
        [Required(ErrorMessage = "поле Популярность обязательное")]
        [StringLength(40, MinimumLength = 2, ErrorMessage = "длина от 2 до 40 симовлов")]
        public string Popularity
        {
            get { return popularity; }
            set { popularity = value; }
        }
    }
}

File: UnionPressOnC/UnionPressOnC/Forms/Models/PeriodModel.cpp
#include <string>
using namespace System::ComponentModel;
using namespace System::ComponentModel::DataAnnotations;
using namespace std;


namespace UnionPressOnC {

	public class PeriodModel
	{
    public:

		string srokSubscribe, sell, transferChoice, popularity;

    public: 
        
        string getSrokSubscribe()
        {
            return srokSubscribe;
        }

        string setSrokSubscribe(string value)
        {
            return srokSubscribe = value;
        }

        string getSell()
        {
            return sell;
        }

        string setSell(string value)
        {
            return sell = value;
        }

        string getTransferChoice()
        {
            return transferChoice;
        }

        string setTransferChoice(string value)
        {
            return transferChoice = value;
        }

        string getPopularity()
        {
            return popularity;
        }

        string setPopularity(string value)
        {
            return popularity = value;
        }

	};
}